### DISCUSSION TITLE
Does RGT see the heart rate?

### DISCUSSION THREAD

**Discussion by arpruss** [doudar/SmartSpin2k#452] [Q&A]
I am wondering if Wahoo RGT, either on PC or Android, can see the heart rate from this device? I made a somewhat similar but less sophisticated device, and Wahoo RGT seems not to see the heart data that is included in the FTMS indoor bike characteristic. (Similarly, if I provide discrete power and heart rate services, it sees the power service but not the heart service, but if I provide just the discrete heart rate service, then it sees it.) Zwift sees it just fine.

**Comment by doudar** [comment]
Same here - I don't think RGT supports reading HRM over FTMS.

**Comment by arpruss** [comment]
Have you contacted Wahoo support? I did, and they said they'd pass it on to their devs, but that they might not do anything because my device is a one-off. You guys on the other hand have a number of users so you might be able to get some action. I assume it's just some assumption in the code that heart rate doesn't come along with any other data.

My current planned workaround is a relay python script on the PC, and then the remote app thinks the PC is a standalone HR device.

**Reply by doudar** [reply]
No, I haven't talked to them about this issue. Feel free to reference it with them if you'd like to take this on! 
Thanks!
